database/meta.go

```diff
@@ -165,6 +166,7 @@ func (m *MetaDBHandle) MarkDBCompleted(name string) error {
 
 }
 
+//UNUSED
 // GetUnAnalyzedDatabases builds a list of database names which have yet to be analyzed and returns
 func (m *MetaDBHandle) GetUnAnalysedDatabases() []string {
```

Comment:
This seems too useful to straight up delete. I marked it as UNUSED. Do a grep for UNUSED to find other functions like this